docs-v2/src/components/TweetClient.tsx

```
@@ -0,0 +1,179 @@
+              rel="noopener noreferrer"
+              className="text-sm font-normal text-gray-500"
+            >
+              <span>{entity.text}</span>
+            </a>
+          );
+        case "text":
+          return (
+            <span
+              key={idx}
+              className="text-sm font-normal"
+              dangerouslySetInnerHTML={{ __html: entity.text }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)